coco/master.py

```diff
@@ -84,6 +94,7 @@ def __init__(self, config_path):
         self.slacker = SlackExporter(self.slack_url)
         config["endpoints"] = self._load_endpoints()
         self._register_config(config)
+        self._start_scheduler()
```

Comment:
Can starting the scheduler here lead to it calling endpoints already? Maybe it should be started last then.

Reply:
yeah, good point, I will change that.

Reply:
`_start_server` is blocking, so the best I can do is start the scheduler just before then. It is set up not to call until one period has elapsed, so maybe this won't be a problem.

Reply:
Sounds good.